d2/src/main/java/com/linkedin/d2/xds/XdsToD2PropertiesAdaptor.java

```diff
@@ -225,6 +224,12 @@ public void onChanged(XdsClient.NodeUpdate update)
         }
       }
 
+      @Override
+      public void onDelete(String name)
```

Comment:
looks like these delete methods are defined and never called? (which is the correct behavior) Would be good to add some comments about why they should never be called.

Reply:
I thought we _were_ going to respect the deletions, so that the future gets resolved faster instead of timing out for 5 seconds?